xAPI.md

```diff
@@ -2483,24 +2483,12 @@ See <a href="#AppendixE">Appendix E: Example Signed Statement</a> for an example
 
 ## 3.0 Metadata
 
-### 3.1 IRI Requirements
-
-###### Metadata Requirements
```

Comment:
It was originally duplicated in this section as a part of the restructuring, decided instead to reference it in the Activity Definition section (link at the end of the paragraph)